README.md

```diff
@@ -51,7 +84,7 @@ If you choose to install dependencies manually as per the instructions above, yo
 
 ## Troubleshooting App Issues
 
-The app is set to create DEBUG logs in the following location: `~/Library/Logs/com.anojht.opensourcesynologycloudsyncdecrypttool.log`
+The app is set to create DEBUG logs in the following location: `~/Library/Logs/com.anojht.opensourcesynologycloudsyncdecrypttool.log or ~/com.anojht.opensourcesynologycloudsyncdecyrpttool.log in the case of Linux.`
```

Comment:
Here the name of the log file should be changed to reflect the changes you made to the name of the file in the code.